### PR TITLE
Add View Caching for Districts

### PR DESCRIPTION
**Pull request by timmyjose**
Chore: https://www.pivotaltracker.com/story/show/166151728

### CODE REVIEW

File: spec/models/call_log_spec.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 require 'rails_helper'
 
 RSpec.describe CallLog, type: :model do
-  describe 'Associations' do
-    it { should belong_to(:user) }
+  describe 'Validations' do
```
Comment (timcheadle) [inline spec/models/call_log_spec.rb]: I think this is unrelated to this PR?

Comment (timmyjose) [reply spec/models/call_log_spec.rb]: @timcheadle You are correct - this is unrelated, but these specs were failing so fixed this in the same PR.

Comment (timmyjose) [reply spec/models/call_log_spec.rb]: @timcheadle As discussed on Slack, moved out the specs changes into a a separate PR, and keeping only relevant changes in this PR. 👍
